src/components/TableComponents/CellTextItem/CellTextItem.tsx

```diff
@@ -0,0 +1,17 @@
+import { CellTextBox, CellTextMain, CellTextAdd } from "./styled";
+
+interface ICellTextItemProps {
+  main: string;
+  additional: string;
+}
+
+const CellTextItem = ({ main, additional }: ICellTextItemProps) => {
+  return (
+    <CellTextBox>
+      <CellTextMain>{main}</CellTextMain>
+      <CellTextAdd>{additional}</CellTextAdd>
```

Comment:
its not a convenient naming, usually devs are using `title` and `info` for such cases or `label` and `info`

Reply:
Fixed)